src/makielayout/blocks/polaraxis.jl

```diff
@@ -0,0 +1,537 @@
+# First, define the polar-to-cartesian transformation as a Makie transformation
+# which is fully compliant with the interface
+
+struct PolarAxisTransformation
+    θ_0::Float64
+    direction::Int
+end
+
+Base.broadcastable(x::PolarAxisTransformation) = (x,)
+
+# original implementation - somewhat inflexible
+# this defined inside initialize_block! for PolarAxis
+# PolarAxisTransformation = Makie.PointTrans{2}() do point
+#     #@assert point isa Point2
+#     y, x = point[1] .* sincos((point[2] + trans.θ_0) * trans.direction)
+#     return Point2f(x, y)
+# end
```

Comment:
Can be removed?

There's also more commented out code in this file that can probably be removed.

Reply:
Yeah, probably could - just wanted to keep a note of other possible approaches.  I'll move all of these to the end of the file.